MatrixSDK/JSONModels/Aggregations/MXAggregationPaginatedResponse_Private.h

```diff
@@ -0,0 +1,30 @@
+/*
+ Copyright 2020 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ The `MXAggregationPaginatedResponse` extension exposes internal operations.
+ */
+@interface MXAggregationPaginatedResponse()
+
+- (instancetype)initWithOriginalEvent:(nullable MXEvent*)originalEvent
```

Comment:
`originalEvent` should be nonnull.
The fact it is null from server response is a synapse bug.